src/sql/workbench/api/common/sqlExtHost.api.impl.ts

```diff
@@ -215,6 +220,14 @@ export function createAdsApiFactory(accessor: ServicesAccessor): IAdsExtensionAp
 				}
 			};
 
+			// namespace: diagnostics
+			const diagnostics: typeof azdata.diagnostics = {
+				// "azdata" API definition
```

Comment:
the newly added comments can be removed as they don't provide any useful information.